cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
Passing -1 isn't an issue. Passing -1, while telling the user that the default value is going to be 10 but then not guaranteeing that seems off. Since snapctl does not control the default value I feel like giving a specific value here without enforcing it could potentially mislead the user.